insights/client/connection.py

```diff
@@ -626,9 +626,6 @@ def register(self):
         """
         Register this machine
         """
-
-        write_to_disk(constants.unregistered_file, delete=True)
-
```

Comment:
I take it this can be removed because you're doing it in `write_registered_file`?

Reply:
Yep